apps/etterlatte-behandling/src/main/kotlin/behandling/etteroppgjoer/EtteroppgjoerRoutes.kt

```diff
@@ -76,11 +76,23 @@ fun Route.etteroppgjoerRoutes(
                 }
             }
 
+            get("hent_faktisk_inntekt") {
```

Comment:
Litt flisespikking her, men tror vi burde holde oss til `-` i stedet for `_` i url'er. Er det som er vanligst. Kan sikkert sløyfe `hent` også siden det er `get`.